script.js

```diff
@@ -1,4 +1,188 @@
+// Temporary array to store individual dice rolls 1 & 2 for each round
+var diceArray = [];
+// Array to collect players' scores for all rounds
+var player1_score = [];
+var player2_score = [];
+// Variable for game modes
+var gameMode = "player 1";
+// Variable for default output value
+var myOutputValue = "Hello, this game is for 2 players. Player 1 and 2 will take turns. Player 1 goes first - click submit to roll 2 dice.";
+
+
+var rollDice = function(i) {
+  var randNum = Math.random() * (i+1);
+  var randNumber = Math.floor(randNum);
+  return randNumber;
```

Comment:
better to be more specific when naming your function parameters 

i should be calld maxDiceNum or something along those lines